pkg/apic/auth/apicauth.go

```diff
@@ -333,10 +336,9 @@ func (ptg *platformTokenGenerator) getPlatformTokens(requestToken string) (*axwa
 
 	defer closeHelper(resp.Body)
 	if resp.StatusCode != 200 {
-		body, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			log.Debugf("bad response from AxwayID: %s: %s", resp.Status, body)
-		}
+		body, _ := ioutil.ReadAll(resp.Body)
+		log.Debugf("bad response from AxwayID: %s: %s, request token :%s, request time : %s", resp.Status, body, requestToken, startTime.String())
```

Comment:
Do we want to log the token here, is that not a security issue?

Reply:
oops... you're right.  this is left over from Peter's binary... thanks!